gsoc_final_post/inigo_tellaetxe_gsoc_2024_final_report.rst

```diff
@@ -0,0 +1,169 @@
+.. image:: https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-logo-horizontal.svg
+   :height: 40
+   :target: https://summerofcode.withgoogle.com/programs/2024/projects/BpY78ovV
+
+.. image:: https://www.python.org/static/community_logos/python-logo.png
+   :height: 40
+   :target: https://summerofcode.withgoogle.com/programs/2024/organizations/python-software-foundation
+
+.. image:: https://python-gsoc.org/logos/DIPY.png
+   :width: 40
+   :target: http://dipy.org
+
+
+Google Summer of Code Final Work Product
+========================================
+
+.. post:: August 21 2024
+   :author: Iñigo Tellaetxe Elorriaga
+   :tags: google
+   :category: gsoc
+
+-  **Name:** Iñigo Tellaetxe Elorriaga
+-  **Organization:** Python Software Foundation
+-  **Sub-Organization:** DIPY
+-  **Project:** `AI/ML in diffusion MRI processing <https://github.com/dipy/dipy/wiki/Google-Summer-of-Code-2024#project-5-project-ideas-using-aiml-in-diffusion-mri-processing>`_
+-  **Repository:** `TractoEncoder GSoC <https://github.com/itellaetxe/tractoencoder_gsoc>`_
+
+
+Abstract
+--------
+
+The objective of the project is to generate synthetic human tractograms with tuneable age and neurodegeneration status properties, using deep learning models. The project is inspired by the `GESTA <https://doi.org/10.1016/j.media.2023.102761>`_ paper, which proposes a generative method using an AutoEncoder architecture for unconditional tractogram generation, combined with a robust sampling strategy. In our case, we replicated the results of the GESTA paper AutoEncoder in the FiberCup dataset using TensorFlow2 as the deep learning backend, instead of the original PyTorch backend. We also investigated how to condition the AutoEncoder architecture on scalar and categorical variables to support conditional tractogram generation on age and neurodegeneration status. Finally, we also explored the use of Variational AutoEncoders, Regression AutoEncoders, and Adversarial AutoEncoders for tractogram generation under the mentioned conditions.
```

Comment:
In our case, we replicated the results of the GESTA paper AutoEncoder in the FiberCup dataset using *TensorFlow2 as the deep learning backend, instead of the original PyTorch backend.*

I would reword this: we did not replicate results. Not sure how you performed the generated of streamlines with different lengths in https://github.com/dipy/dipy.org/pull/63/files, but I do not recall having discussed actually having reached the point where the work on the AE would enable us to move to the sampling part.

Reply:
We replicated the FINTA AutoEncoder, that is what I wanted to say. I mixed it up sorry. No GESTA sampling